text/0156-columnar-encoding.md

```diff
@@ -0,0 +1,151 @@
+# Columnar encoding for the Open Telemetry protocol
```

Comment:
```suggestion
# Columnar encoding for the OpenTelemetry protocol
```

one word

Reply:
Updated rest of the document accordingly to this edit.